### DISCUSSION TITLE
How to set a default model?

### DISCUSSION THREAD

**Discussion by AumCoin** [danny-avila/LibreChat#4865] [Help Wanted]
I am running LibreChat in docker.  I browse to http://myserver:3080/c/new and I would like it to always start with OpenAI:gpt-4o-mini.  I find that it does this until it requires me to log in again (because the solution for that doesn't work consistently somehow), and then it resets to 4o (a more expensive model).  Is there a way I can just specify a default model that will always be chosen automatically when a user logs in?

**Comment by m0eak** [comment]
Same question
